modules/ROOT/pages/sso-use-case-1.adoc

```diff
@@ -59,7 +66,7 @@ You set up the auth provider or SSO settings in Salesforce with the identity pro
 .. In *Salesforce*, go to *Setup*.
 .. In the *Quick Find* box, enter `Auth`, and then select *Auth. Providers*.
 .. Click *New*.
-.. In the Provider Type field, select *OpenID Connect*.
+.. In the *Provider Type* field, select *OpenID Connect*.
```

Comment:
Is this a menu? Maybe just remove say "For *Provider Type*, select"?

Reply:
done